test/.zattrs

```diff
@@ -0,0 +1,117 @@
+{
+    "multiscales": [
```

Comment:
I think these generated files can be removed from the git repo. Feel free to add to `.gitignore` if it's more convenient.

Reply:
I'm not quite sure what proper steps to take to remove the files. Could we discuss this and .gitignore further?

Reply:
From the command line, you can stop tracking a file with `git rm file.txt` then `git commit`. It's typically `right-click -> delete` in git GUIs. 

You can also add a specific file name or glob to the `.gitignore` file to ignore a file.

It's a bit tricky. We can chat later this morning if you can't get it to work.